CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+## Sending a pull request
+Please test the **Legofy** script first before sending a pull request to make sure you didn't mess up
```

Comment:
Please elaborate.
- Fork
- Create feature branch
- Add tests for new feature
- Make sure Travis CI passes (You can activate Travis CI for your own fork)
- Make sure coverage does not lower
- (Merge commits into one)
- Send pull request